### PR TITLE
PR from i4song-branch to master

### PR DESCRIPTION
**Pull request by o-bard-o**
## What is this PR?

- Create `i4song/index.js`

### CODE REVIEW

File: i4song/index.js [hunk 1]
```diff
@@ -0,0 +1 @@
+console.log("Hello World!");
```
Comment (o-bard-o) [inline i4song/index.js:1]: Hello World! 를 출력합니다.

Comment (hantaihe) [reply i4song/index.js:1]: Good code!

**Comment by hantaihe** [comment]
👍👍👍

**Review by hantaihe — APPROVED** [review]
LGTM

**Pull request merged**